test/disable_tf_files.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+function org(){
+    # disable backend configs in main module
+    mv 1-org/envs/shared/backend.tf 1-org/envs/shared/backend.tf.disabled
+}
+
+function envs(){
+    # disable backend configs in main module
+    mv 2-environments/envs/development/backend.tf 2-environments/envs/development/backend.tf.disabled
+    mv 2-environments/envs/non-production/backend.tf  2-environments/envs/non-production/backend.tf.disabled
+    mv 2-environments/envs/production/backend.tf  2-environments/envs/production/backend.tf.disabled
+}
+
+function networks(){
+    # disable backend configs in main module
+    mv 3-networks/envs/development/backend.tf 3-networks/envs/development/backend.tf.disabled
+    mv 3-networks/envs/non-production/backend.tf  3-networks/envs/non-production/backend.tf.disabled
+    mv 3-networks/envs/production/backend.tf  3-networks/envs/production/backend.tf.disabled
+
+    # disable access_context.auto.tfvars in main module
```

Comment:
Lets open an issue in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit to add a helper or design a pattern for this.

Reply:
Issue created

https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1124